# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/irda-utils/Makefile-irdadump.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile	2004-11-10 18:35:54.911504944 +0100
+++ Makefile-p	2004-11-10 18:38:24.441772888 +0100
@@ -33,7 +33,7 @@
 	mkdir -p $(RPM_BUILD_ROOT)/usr/bin $(RPM_BUILD_ROOT)/usr/X11R6/bin
 ## Red-Hat specific, not available on Debian
 ##	mkdir -p $(RPM_BUILD_ROOT)/etc/rc.d/init.d
-	cd irdadump;CFLAGS="$(CFLAGS)" CXXFLAGS="$(CFLAGS)" ./autogen.sh --prefix=$(RPM_BUILD_ROOT)/usr
+	cd irdadump;./configure --prefix=/usr --bindir=/usr/sbin
 	for i in $(DIRS); do \
                 cd $$i;make all; cd ..; \
         done
